lib/internal/process/task_queues.js

```diff
@@ -160,13 +205,23 @@ function queueMicrotask(callback) {
 
 module.exports = {
   setupTaskQueue() {
+    if (experimentalTaskOrdering === undefined) {
+      const { getOptionValue } = require('internal/options');
+      experimentalTaskOrdering = getOptionValue('--experimental-task-ordering');
+    }
+
     // Sets the per-isolate promise rejection callback
     listenForRejections();
     // Sets the callback to be run in every tick.
-    setTickCallback(processTicksAndRejections);
+    setTickCallback(experimentalTaskOrdering
+      ? processTicksAndRejectionsNew
+      : processTicksAndRejectionsOld
+    );
```

Comment:
nit: fix lint errors
```suggestion
    setTickCallback(experimentalTaskOrdering ?
      processTicksAndRejectionsNew :
      processTicksAndRejectionsOld);
```